packages/client/package.json

```diff
@@ -83,6 +83,26 @@
       "import": "./wasm.js",
       "default": "./wasm.js"
     },
+    "./runtime/library": {
+      "require": "./runtime/library.js",
+      "import": "./runtime/library.js",
+      "default": "./runtime/library.js"
+    },
+    "./runtime/binary": {
+      "require": "./runtime/binary.js",
+      "import": "./runtime/binary.js",
+      "default": "./runtime/binary.js"
+    },
+    "./runtime/edge": {
+      "require": "./runtime/edge.js",
+      "import": "./runtime/edge.js",
+      "default": "./runtime/edge.js"
+    },
+    "./runtime/wasm": {
+      "require": "./runtime/wasm.js",
+      "import": "./runtime/wasm.js",
+      "default": "./runtime/wasm.js"
+    },
```

Comment:
I initially had a more compact version, which turned out to be a time sink to make it work with Next.js 13 (misinterpreted), so I opted towards a more verbose and more archaic version of it.

```ts
    "./*.js": "./*.js",
    "./*.d.ts": "./*.d.ts",
    "./*": {
      "types": "./*.d.ts",
      "require": "./*.js",
      "import": "./*.js",
      "default": "./*.js"
    }
```